### PR TITLE
Ref#43812 make auth seeder

### PR DESCRIPTION
**Pull request by hoangtrungthong**
## Related Tickets:
- [#43812](https://edu-redmine.sun-asterisk.vn/issues/43812)

## WHAT this PR does?
make auth seeder

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by hoangtrungthong** [comment]
ready

File: database/seeders/RolesTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     *
+     * @return void
+     */
+    public function run()
+    {
+        Role::create([
+            'name' => 'admin',
+        ]);
+        Role::create([
+            'name' => 'user',
```
Comment (ghost) [inline database/seeders/RolesTableSeeder.php]: bn bỏ thông tin setting role vào config nha

Comment (hoangtrungthong) [reply database/seeders/RolesTableSeeder.php]: ok bạn

**Comment by hoangtrungthong** [comment]
ready

**Review by ghost — APPROVED** [review]

File: database/seeders/UsersTableSeeder.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public function run()
+    {
+        User::create([
+            'name' => 'admin',
+            'email' => 'admin@mobile.com',
+            'phone' => '0123456789',
+            'address' => 'Co nhue',
+            'password' => Hash::make('123123123'),
+            'role_id' => config('const.admin'),
+        ]);
```
Comment (ghost) [inline database/seeders/UsersTableSeeder.php]: admin data bỏ zô config nha :)

**Comment by hoangtrungthong** [comment]
ready

**Pull request merged**